Starting /home/h/hc443/.bash_profile ... standard AFS bash profile

========================
Home directory : /home/h/hc443 is not in AFS -- skipping quota check
========================

On host node436 :
	 19:44:36 up 72 days, 23:43,  0 users,  load average: 1.00, 1.08, 0.96

=== === === Your Kerberos ticket and AFS token status === === ===
klist: No credentials cache found (filename: /tmp/krb5cc_515598)
Kerberos : 
AFS      : 

Loading default modules ...
Create file : "/home/h/hc443/.modules" to customize.

No modules loaded
Traceback (most recent call last):
  File "/home/h/hc443/bench_dev/BGPM/run_model.py", line 36, in <module>
    run_model(task=args.task, model_name=args.model, dataset_name=args.dataset,
  File "/home/h/hc443/bench_dev/BGPM/libgptb/pipeline/pipeline.py", line 24, in run_model
    config = ConfigParser(task, model_name, dataset_name,
  File "/home/h/hc443/bench_dev/BGPM/libgptb/config/config_parser.py", line 26, in __init__
    self._load_default_config() 
  File "/home/h/hc443/bench_dev/BGPM/libgptb/config/config_parser.py", line 77, in _load_default_config
    raise ValueError('task {} do not support model {}'.format(
ValueError: task GCL do not support model GPT
srun: error: node436: task 0: Exited with exit code 1
